command/main.go

```diff
@@ -23,7 +48,83 @@ func Run(args []string) int {
 			args = []string{"version"}
 			break
 		}
+
+		// Parse a given flag here, which overrides the env var
+		if strings.HasPrefix(arg, "-format=") {
+			format = strings.TrimPrefix(arg, "-format=")
+		}
+		// For backwards compat, it could be specified without an equal sign
+		if arg == "-format" {
+			nextArgFormat = true
+		}
+	}
+
+	envVaultFormat := os.Getenv(EnvVaultFormat)
+	// If we did not parse a value, fetch the env var
+	if format == "" && envVaultFormat != "" {
+		format = envVaultFormat
```

Comment:
We can probably do `format = strings.ToLower(envVaultFormat)` directly.

Reply:
We'd still need it outside the `if` block so it doesn't buy us anything.